ansible/deploy-openstack-config.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Deploy OpenStack Configuration
+  hosts: all
+  gather_facts: true
+  vars_files:
+    - vars/defaults.yml
+  roles:
+    - { name: mrlesmithjr.manage-lvm, become: true }
```

Comment:
I'd suggest making this a separate playbook. It won't always be necessary

Reply:
Could this be potential solution. Rather than creating another playbook. Instead use the `never` tag to prevent it from running by default. If you require LVMs to be resized then you can use `--tags=lvm`. Added to documentation.
```  
roles:
    - { name: mrlesmithjr.manage-lvm, become: true, tags: [never, lvm] }
```